setup.py

```diff
@@ -1,11 +1,11 @@
 from setuptools import setup
 
 setup(
-    name='RateMyPDF',
-    version='1.0',
+    name="RateMyPDF",
+    version="1.0",
     long_description=__doc__,
-    packages=['app'],
+    packages=["app"],
     include_package_data=True,
     zip_safe=False,
-    install_requires=['Flask','FormFyxer']
-)
\ No newline at end of file
+    install_requires=["Flask", "FormFyxer"],
```

Comment:
FormFyxer needs a release to include https://github.com/SuffolkLITLab/FormFyxer/pull/101, and this needs to be updated to force that new version. Won't work in prod without that.